.ide-toolbox.metadata.json

```diff
@@ -0,0 +1,199 @@
+{
+    "registrar":[
+        {
+            "provider":"contao_system_classes",
+            "language":"php",
+            "signature":[
+                "Contao\\CoreBundle\\Framework\\ContaoFrameworkInterface:getAdapter",
+                "Contao\\CoreBundle\\Framework\\ContaoFrameworkInterface:createInstance"
+            ],
+            "signatures":[
+                {
+                    "class": "Contao\\CoreBundle\\Framework\\ContaoFrameworkInterface",
+                    "method": "getAdapter",
+                    "type": "type"
+                },
+                {
+                    "class": "Contao\\CoreBundle\\Framework\\ContaoFrameworkInterface",
+                    "method": "createInstance",
+                    "type": "type"
+                }
+            ]
+        },
+        {
+            "provider":"contao_model_classes",
+            "language":"php",
+            "signature":[
+                "Contao\\CoreBundle\\Framework\\ContaoFrameworkInterface:getAdapter"
+            ],
+            "signatures":[
+                {
+                    "class": "Contao\\CoreBundle\\Framework\\ContaoFrameworkInterface",
+                    "method": "getAdapter",
+                    "type": "type"
+                }
+            ]
+        },
+        {
+            "provider":"contao_framework_classes",
+            "language":"php",
+            "signature":[
+                "Contao\\CoreBundle\\Framework\\ContaoFrameworkInterface:getAdapter"
```

Comment:
The provider is different. The first one allows subclasses of `Contao\System`, but some are not (like `System` itself and `Config`, maybe there's even more).